data/korath/korath ships.txt

```diff
@@ -270,6 +270,78 @@ ship "Korath Raider Mark XLII"
 	description `Constant warfare against surrounding aliens has been building pressure on Korath shipyards to recover some of their ancient prowess. Now, with a fresh industrial network to support their endeavors, the shipwrights of the Korath have updated the design to deal with modern targets.`
 
 
+ship "Korath Ravager"
```

Comment:
Question: The name here is "Korath Ravager" and the description calls it a "'nra'ret-Class Chaser." I think I must have missed the discussion regarding its name.

Shouldn't this be consistent one way or the other (preferably its actual Korath name, rather than having the Korath prefix)